WTWSMS/common/objectives/00_focuses.txt

```diff
@@ -209,7 +214,7 @@ focus_seduction = {
 		}
 		
 		modifier = {
-			factor = 0.01 # Faithful spouse/consort
+			factor = 0 # Faithful spouse/consort
```

Comment:
Small factors will lead to the AI picking it as some point as it constantly re-evaluates it. Agressive seduction can be enabled by gamerule.